src/data_structures/interpolation_table.rs

```diff
@@ -3,15 +3,17 @@
 // This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! # Easy Example
+//! Fast lookup table for arbitrary floating point functions.
+//! # Examples
+//! ## Easy:
 //! ```
 //! use bio::data_structures::interpolation_table::*;
 //!
 //! let table = InterpolationTable::new(0.0, 10.0, 5, |x| x.powf(2.0));
 //! assert_eq!(table.get(3.0), 9.0);
 //! assert_eq!(table.get(5.0), 25.0);
 //! ```
-//! # More complicated
+//! ## More complicated:
 //! ```
 //! extern crate approx;
```

Comment:
```suggestion
```

Reply:
Ah, whatever, removing the unnecessary `extern crate` (rust-bio uses edition 2018) can be done later for all the doc code examples in batch